### ISSUE TITLE
Scala support library for RestAssured is not cross compiled

### ISSUE THREAD

**Issue by xeppaka** [rest-assured/rest-assured#1211]
Shouldn't scala support library be cross compiled? We use scala 2.12.9 and as the latest rest assured (v4.1.1) scala-support is compiled with scala 2.13 we can't use it in the project.

**Comment by johanhaleby** [comment]
@xeppaka Yes it probably should! Can you do this easily with maven? Care to provide a PR if you know how to do it?

**Comment by xeppaka** [comment]
> @xeppaka Yes it probably should! Can you do this easily with maven? Care to provide a PR if you know how to do it?

Don't know how it's done with maven... I'll try to find out
